integration/src/test/java/pages/CapabilitiesPage.java

```diff
@@ -0,0 +1,43 @@
+package pages;
+
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.WebElement;
+import org.openqa.selenium.support.FindBy;
+
+import java.util.List;
+
+public class CapabilitiesPage extends PageObject{
+
+    @FindBy(id = "capTitle") private WebElement Title;
+
+    @FindBy(id = "capName") private List<WebElement> capabilities;
+
+    public CapabilitiesPage(WebDriver driver) {
+        super(driver);
+    }
+
+    public String getTitle() {
+        System.out.println(Title.getText());
+        return Title.getText(); }
+
+    public boolean isCapability (String capability) {
+        for (WebElement cap : capabilities)
+        {
+            if (cap.getText().equals(capability)) {
+                return true;
+            }
+        }
+        return false;
```

Comment:
First, there's some inconsistent formatting here
```suggestion
        for (WebElement cap : capabilities) {
            if (cap.getText().equals(capability)) {
                return true;
            }
        }
        return false;
```

Second, streams will simplify this logic
```suggestion
        return capabilities.stream()
            .map(Capability::getText)
            .anyMatch(capability::equals);
```

Reply:
See with how you are trying to simplify the logic. When trying to map to the Capability Class it complains no instances have been created.